tests/robustness/validate/patch_history.go

```diff
@@ -29,7 +30,9 @@ func patchLinearizableOperations(reports []report.ClientReport, persistedRequest
 	persistedPutCount := countPersistedPuts(persistedRequests)
 	clientPutCount := countClientPuts(reports)
 	putReturnTime := uniquePutReturnTime(allOperations, reports, persistedRequests, clientPutCount)
-	return patchOperations(allOperations, putRevision, putReturnTime, clientPutCount, persistedPutCount)
+	patchedOperations := patchOperations(allOperations, putRevision, putReturnTime, clientPutCount, persistedPutCount)
```

Comment:
Please don't do that for patching, but for validation. This is required part of validation due to how pourcupine work, while patching is just optimization. I would want to have it separate process before patching as part of validation.

Reply:
Moved into the validation function